po/fr.po

```diff
@@ -240,8 +240,9 @@ msgid "Lifetimes in Data Structures"
 msgstr "Durées de vie dans les structures de données"
 
 #: src/SUMMARY.md:67
-msgid "Designing a Library"
-msgstr "Conception d'une bibliothèque"
+#, fuzzy
+msgid "Storing Books"
+msgstr "String"
```

Comment:
This is really weird. Where does the translation come from? I did not change anything after running msgmerge. (there are a couple of those), just leaving them for a second, in case the reviewer knows what's going on.

Reply:
I see the "Desigining a Library" string was moved at the end of the file, in a commented string?

Reply:
Hi @yohcop, what you see here is `msgmerge` looking at the old and new strings and trying it's best (it's not very clever) to find an existing translation for the new changed `msgid`.

As you know, the `msgid` fields change as we evolve the English source text. So when we fix a typo (#916), the `msgid` changes when you regenerate `messages.pot`. When you run `msgmerge` the `msgid` in your `fr.po` file is compared with the ones in `messages.pot` and here `msgmerge` was able to correctly match up the updated English text with the existing French translations on [line 10,920](https://github.com/google/comprehensive-rust/pull/918/files#diff-940623802973a9e80a24d1c2e258a73d0e9c532b9feb52d2607681dca35f83c6L10918-R10994).

For a typo fix, the French translation will still work (unless someone copied the typo when translating :smile:). All that's left to do is to remove the `fuzzy` marker that `msgmerge` inserts. This marker says "The translation here is _fuzzy_ and needs a review." We don't use fuzzy translations at all, so it's important that you do a sweep of the `fr.po` file afterwards to clean them up.

In this case, you see the strange guess is marked fuzzy as well. So it's weird, but ultimately harmless.

Finally, what you see at the bottom is a kind of recycle bin for old translations — translations that were made obsolete since their `msgid` no longer matches any `msgid` in the `messages.pot` file:

```po
#~ msgid "Designing a Library"
#~ msgstr "Conception d'une bibliothèque"

#, fuzzy
#~ msgid "# Designing a Library"
#~ msgstr "# Concevoir une bibliothèque"
```

(cc @henrif75 since this is good stuff to know when dealing with translations)